From 11cbc7044e8999212e02b3665f2b7d10a26f939a Mon Sep 17 00:00:00 2001
From: Chen Lin Z <lin.z.chen@intel.com>
Date: Mon, 15 Oct 2018 16:34:41 +0800
Subject: [PATCH] Allow init to create tmpfs lnk_file

{ create } for pid=1 comm="init" name="sdcard" scontext=u:r:init:s0
tcontext=u:object_r:tmpfs:s0 tclass=lnk_file permissive=0

Change-Id: I064458de3e757ef6a5970a371c59a4468c55d0e2
Tracked-On:
Signed-off-by: Chen Lin Z <lin.z.chen@intel.com>
---
 public/init.te | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/public/init.te b/public/init.te
index dafc06f99..8e2582910 100644
--- a/public/init.te
+++ b/public/init.te
@@ -38,7 +38,7 @@ allow init block_device:{ dir blk_file lnk_file } relabelto;
 allow init dm_device:{ chr_file blk_file } relabelto;
 allow init kernel:fd use;
 # restorecon for early mount device symlinks
-allow init tmpfs:lnk_file { getattr read relabelfrom };
+allow init tmpfs:lnk_file { getattr read relabelfrom create };
 allow init {
   misc_block_device
   recovery_block_device
-- 
2.22.0

